### PR TITLE
Cat 1439 Fix Returning to ScriptActivity when Dismissing new Look Dialog

### PR DESCRIPTION
**Pull request by loibi93**
https://jenkins.catrob.at/view/All-Categories/view/Catroid-multi-job/job/Catroid-Multi-Job-Custom-Branch-RELOADED/2058/

https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-CustomEmulator-Single-ClassAndPackage-Test/398/

Warnings are only for Multiline !

### CODE REVIEW

**Comment by thmq** [comment]
commits should be squashed into one.

**Comment by loibi93** [comment]
Whats the best way to do this ? ;D
git rebase -i develop ?

**Comment by thmq** [comment]
yes, most likely.

**Comment by loibi93** [comment]
Done ! Automatic Tests from Jenkins are failing, but as you can see above everything was ok ! So if the Code is ok this can be merged ;)

File: catroid/src/org/catrobat/catroid/ui/fragment/LookFragment.java [hunk 1]
```diff
@@ -587,12 +588,24 @@ private void handleAddButtonFromNew() {
 		}
 	}
 
+
```
Comment (obusher) [inline catroid/src/org/catrobat/catroid/ui/fragment/LookFragment.java]: O_O!

File: catroid/src/org/catrobat/catroid/ProjectManager.java [hunk 2]
```diff
@@ -29,6 +29,7 @@
 import android.support.v4.app.FragmentActivity;
 import android.util.Log;
 
+
 import org.catrobat.catroid.common.Constants;
```
Comment (obusher) [inline catroid/src/org/catrobat/catroid/ProjectManager.java]: O_O!

File: catroid/src/org/catrobat/catroid/ProjectManager.java [hunk 3]
```diff
@@ -29,6 +29,7 @@
 import android.support.v4.app.FragmentActivity;
 import android.util.Log;
 
+
```
Comment (obusher) [inline catroid/src/org/catrobat/catroid/ProjectManager.java]: :O

Comment (loibi93) [inline catroid/src/org/catrobat/catroid/ProjectManager.java]: wait a second xD

**Comment by obusher** [comment]
https://jenkins.catrob.at/job/Catroid-Build-SourceTests-StaticChecks/2689/checkstyleResult/new/

**Comment by thmq** [comment]
ok to test

**Comment by obusher** [comment]
ok to test

**Comment by obusher** [comment]
ok to test

**Comment by loibi93** [comment]
Only one test failed, seems to be a jenkins bug, i run it again and it didn't fail:
https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-Device-Single-ClassAndPackage-Test/318/

**Comment by obusher** [comment]
It works but you can do better! :D

As mentioned in [CAT-1439](https://jira.catrob.at/browse/CAT-1439) i would not even bother to change to the `LookFragment`. Do some modifications in the `NewLookDialog` so you can call this dialog from the `ScriptFragment`.

**Comment by loibi93** [comment]
As you said it works, if you don't want that the LooksFragment is in Background!!! then please open a new Issue for it !

If my Code is OK, please merge !

**Comment by thmq** [comment]
the Jenkins run seems fine to me. depending on how urgent the fix is, creating a new issue might be best (if the fix should be merged soon).

**Comment by loibi93** [comment]
The issue is marked as major, so i suggest merging it, and then discuss with the UX Team if it is even necessary to rewrite this. I think that this solution is better, because it is consistent (the add dialog is always on the corresponding Fragment) and the user has a Feedback that he is adding a Look (or a Sound).

**Comment by loibi93** [comment]
ok to test

**Comment by loibi93** [comment]
ok to test

**Comment by loibi93** [comment]
ok to test

**Comment by loibi93** [comment]
The two tests that are failing have nothing to do with my Code, as I said in IRC I think some tests are faulty, but we can discuss that today at the meeting !

**Comment by avihric** [comment]
ok to test

**Pull request merged**